packages/application-extension/src/index.ts

```diff
@@ -255,7 +255,7 @@ function addCommands(app: JupyterLab, palette: ICommandPalette): void {
   command = CommandIDs.toggleLeftArea;
   app.commands.addCommand(command, {
     label: args => args['isPalette'] ?
-    'Toggle Left Area' : 'Show Left Area',
+    'Toggle Left Sidebar' : 'Show Left Sidebar',
```

Comment:
I thought the left sidebar was either both the vertical tabs and the panel, or maybe just the left panel, whatever is being shown, and we don't use the term "left area" anymore. @ellisonbg, @jzf2101 ?

Reply:
I think @ian-r-rose  is right.

Reply:
If we are looking at other uses of "sidebar", such as OS X apps, the sidebar is the actual content region on the left (and there is no analog to our vertical tabs that always are visible). That's why in OS X, it is literally "Hide Sidebar" in the menu.

Same with OpenOffice and other apps: https://en.wikipedia.org/wiki/Sidebar_(computing) - the sidebar hosts the content:

>  Unlike toolbars and status bars, sidebars have larger surface areas because of horizontally longer layout of desktop apps. Sidebars may use accordions to organize widgets and accommodate a larger layout than the visible surface area.

Typically a sidebar has an accordion to organize sections, but we use vertical tabs. I think widespread accepted usage is that "sidebar" refers to the content area, though. Since we use tabs instead of accordions, it's a bit ambiguous to me whether "sidebar" in our case includes the tabs, or is just the content area (I would probably say it includes the tabs, i.e., is the entire left tab panel, and maybe terminology should be "expand/collapse sidebar").

Apparently there was still some misunderstanding when we defined terms before. @ellisonbg, you were in that discussion before too--what were you thinking "sidebar" meant?

Reply:
I'm willing to wait on @ellisonbg, @tgeorgeux do you have opinions as well?